index.d.ts

```diff
@@ -55,6 +55,26 @@ type Delay = {
 		}
 	): delay.ClearablePromise<T>;
 
+	/**
+	Create a promise which resolves after a random amount of milliseconds between `minimum` and `maximum` has passed.
+
+	Useful for testing and web scraping.
```

Comment:
Useful how? This statement is not very useful as it's missing the **why**. The method is useful because tests and scraping can have unpredictable speed. For example, if you have a test that asserts a method should not take longer than a certain amount, and then run it on a CI, it could take longer. So with `.range()`, you could give it a threshold instead.

Reply:
Fixed!